cmd/osm-bootstrap/crds/config_mesh_root_certificate.yaml

```diff
@@ -37,9 +37,9 @@ spec:
       served: true
       storage: true
       additionalPrinterColumns:
-        - description: Current state of the MeshRootCertificate config
-          jsonPath: .status.state
-          name: State
+        - description: Intent for the MeshRootCertificate
+          jsonPath: .spec.intent
+          name: Intent
```

Comment:
As I have been playing around with the MRC for the e2e tests and been thinking this field name might be better as `Role`. for example:

"Moving MRC to Passive intent in cluster\n" compared to "Moving MRC [%s] -> Passive role cluster"

"The MRC is in the Active role in the cluster" compared to "The MRC is in the Active intent in the cluster"

Thoughts?  @jaellio @keithmattix @steeling

Reply:
ya that sounds good to me.

Reply:
Okay! I will make the update

Reply:
#5222

Reply:
#5222

Reply:
I think Role makes sense since we're ultimately representing what the certificate can and cannot do.